### PR TITLE
refactor(elevation): no default dom mutation

### PR DESCRIPTION
**Pull request by yinonov**

### CODE REVIEW

File: libs/components/src/lib/elevation/README.md [hunk 1]
```diff
@@ -10,96 +10,84 @@ The _elevation_ component enables a user to set perceived elevation to a certain
...
+ }
 </style>
 
-<vwc-elevation id="elevation" dp="0">
-	<div class="card">
-		Hover me!
-	</div>
+<vwc-elevation id="elevation">
+ <div class="card">
+  Hover me!
```
Comment (rinaok) [inline libs/components/src/lib/elevation/README.md:29]: The hover is not working anymore

File: libs/components/src/lib/elevation/README.md [hunk 2]
```diff
@@ -10,96 +10,84 @@ The _elevation_ component enables a user to set perceived elevation to a certain
...
+  </div>
+ </vwc-elevation>
 </vwc-layout>
 ```
 
 ## CSS Custom Properties
+
 | Name                        | Description                                        | Default |
 | --------------------------- | -------------------------------------------------- | ------- |
 | `--elevation-border-radius` | Border radius of the elevation’s surface in pixels | 6px     |
```
Comment (rinaok) [inline libs/components/src/lib/elevation/README.md:93]: You removed this property

Comment (yinonov) [reply libs/components/src/lib/elevation/README.md:107]: it didn't work anyway

Comment (rachelbt) [reply libs/components/src/lib/elevation/README.md]: you removed the` border-radius` in the css so it's not working. But we do need the  `border-radius`

Comment (yinonov) [reply libs/components/src/lib/elevation/README.md]: now that it's applied on its slotted assigned nodes it doesn't
image

Comment (rachelbt) [reply libs/components/src/lib/elevation/README.md]: can reproduce that....
image

File: libs/components/src/lib/elevation/README.md [hunk 3]
```diff
@@ -11,89 +11,87 @@ The _elevation_ component enables a user to set perceived elevation to a certain
...
-		<div class="card">
-			This is the content inside the elevation with DP 0
-		</div>
-	</vwc-elevation>
-	<vwc-elevation dp="2">
-		<div class="card">
-			This is the content inside the elevation with DP 2
-		</div>
-	</vwc-elevation>
+<vwc-layout column-spacing="base-large" auto-sizing="fit" gutters="base-small">
```
Comment (rinaok) [inline libs/components/src/lib/elevation/README.md:60]: @yinonov  The differences between the dps are difficult to see. Why didn't you keep the column-based block basis?

File: libs/components/src/lib/elevation/ui.test.ts [hunk 4]
```diff
@@ -11,6 +11,11 @@ test('should show the component', async ({ page }: { page: Page }) => {
 	const template = extractHTMLBlocksFromReadme(path.join(__dirname, 'README.md'))
 		.reduce((htmlString: string, block: string) => `${htmlString} <div style="margin: 5px;">${block}</div>`, '');
 
+	await loadComponent({
+		page,
+		componentName: 'layout',
```
Comment (rinaok) [inline libs/components/src/lib/elevation/ui.test.ts:16]:
```suggestion
		componentName: 'elevation',
```

File: libs/components/src/lib/elevation/README.md (hunk 3 above)
Comment (yinonov) [reply libs/components/src/lib/elevation/README.md]: restored

**Pull request merged**